src/PostgREST/Config.hs

```diff
@@ -51,6 +52,7 @@ argParser = AppConfig
   <$> argument str (help "(REQUIRED) database connection string, e.g. postgres://user:pass@host:port/db" <> metavar "DB_URL")
   <*> strOption    (long "anonymous"  <> short 'a' <> help "(REQUIRED) postgres role to use for non-authenticated requests" <> metavar "ROLE")
   <*> strOption    (long "schema"     <> short 's' <> help "schema to use for API routes" <> metavar "NAME" <> value "public" <> showDefault)
+  <*> strOption    (long "hostname"   <> short 'n' <> help "hostname on which the HTTP server is running" <> metavar "HOSTNAME" <> value "localhost" <> showDefault)
```

Comment:
i don't think it's a problem, we just need to document exactly what this 'hostname', and make it explicit that it's used when outputing swagger. Maybe it's not even ok to name it hostname, it's more of a url because to postgrest it might be someip:3000 but to outside world, the api i accessed using https:/some domain.com/api/v1/ so actually this parameters shoudl be the full url prefix.
the other thing is almost all the time an IP